pyprophet/classifiers.py

```diff
@@ -110,7 +110,7 @@ def objective(params):
             
             clf = xgb.XGBClassifier(random_state=42, verbosity=0, objective='binary:logitraw', eval_metric='auc', **params)
 
-            score = cross_val_score(clf, X, y, scoring='roc_auc', n_jobs=self.threads, cv=KFold(n_splits=3, shuffle=True, random_state=np.random.RandomState(42))).mean()
+            score = cross_val_score(clf, X, y, scoring='roc_auc', n_jobs=self.threads, cv=KFold(n_splits=3, shuffle=True, random_state=42)).mean()
```

Comment:
It can be either an integrer or a np.random.RandomState however np.random.RandomState is deprecated in newer versions of numpy (or might already be deprecated) so I switched to integer here